install-deps.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -e
+#!/bin/sh -e
```

Comment:
On 22-3-2016 04:07, Kefu Chai wrote:
> In install-deps.sh
> <https://github.com/ceph/ceph/pull/7515#discussion_r56931889>:
> 
>> @@ -1,4 +1,4 @@
>> -#!/bin/bash -e
>> +#!/bin/sh -e
> 
> how's your test going? i see you instruct user to create a symlink in
> README.FreeBSD, so we can dispense with this change?

Nope,

I took this change originally from Mykola and then this change got
"flamed" by Sage, in that /bin/sh in certain platforms turns into
/bin/dash. And that is going to conflict with a lot of bashisms.

So I think this is teh easy way out.
And to be honest, if you want to port any serious software to FreeBSD it
is almost impossible to NOT add this /bin/bash link.

It used to be an option in the package, but recently it has been decided
that port do not "automagically" link/install/change these kind of actions.

So options are:
 - create a link and be done
 - modify all scripts to use /usr/bin/env bash
   and additionally fix any options on the #!-bang line.

--WjW


--WjW